tests/jsunit/xml_test.js

```diff
@@ -46,6 +52,39 @@ var XML_TEXT = ['<xml xmlns="http://www.w3.org/1999/xhtml">',
   '  </block>',
   '</xml>'].join('\n');
 
+
+function xmlTest_setUpWithMockBlocks() {
+  workspace = new Blockly.Workspace();
+  mockControl_ = new goog.testing.MockControl();
+  // Need to define this because field_variable's dropdownCreate() calls replace
+  // on undefined value, Blockly.Msg.DELETE_VARIABLE. To fix this, define
+  // Blockly.Msg.DELETE_VARIABLE as %1 so the replace function finds the %1 it
+  // expects.
+  Blockly.Msg.DELETE_VARIABLE = '%1';
+}
+
+function xmlTest_tearDownWithMockBlocks() {
+  workspace.dispose();
+  mockControl_.$tearDown();
+  delete Blockly.Blocks.field_variable_test_block;
+  Blockly.Msg.DELETE_VARIABLE = saved_msg;
+}
+
+/**
+ * Check the values of the field dom.
+ * @para, {!Element} fieldDom The xml dom of the field.
+ * @param {!string} name The expected name of the variable.
+ * @param {!string} type The expected type of the variable.
+ * @param {!string} id The expected id of the variable.
+ * @param {!string} id The expected text of the variable.
+ */
+function xmlTest_checkFieldDomValues(fieldDom, name, type, id, text) {
```

Comment:
checkVariable or checkFieldVariable -> checkVariableDomValues since this is specific to variable fields and can't be used for other fields.